README.md

```diff
@@ -140,7 +140,7 @@ This repository contains many companion libraries to write integration tests wit
 | Wrap a test group to use a `withResource`-like function (`aroundAll`)                     | ✔️       | ✔️[4]                                                           | ✖️                                                                |
 | Randomising execution order                                                               | ✔️       | ✔️                                                           | C                                                                |
 | Randomised execution order by default                                                     | ✔️       | ✖️                                                           | C                                                                |
-| Deterministic randomness for randomised execution                                         | ✔️       | ✖️                                                           | C                                                                |
+| Deterministic randomness for randomised execution                                         | ✔️       | ✔️                                                           | C                                                                |
```

Comment:
Hspec uses `--seed` for test order randomization, so I think this was always the case; please prove me wrong.